lua/bufferline/utils.lua

```diff
@@ -9,6 +9,7 @@ local fnamemodify = vim.fn.fnamemodify
 local split = vim.split
 local join = table.concat
 local strwidth = vim.api.nvim_strwidth
+local opts = vim.g.bufferline
```

Comment:
Does this work if `g:bufferline` is defined/modified after the script is loaded? I'd rather put it inside `get_buffer_name`, or maybe pass it as an argument to `get_buffer_name`. Not sure of the performance characteristics of property acces across the vimscript/lua boundary.

Reply:
thank you for the advice and this is the change in the code, is it appropriate?


sorry if my english to bad

Reply:
Two more things:
 - Keep the `local opt = vim.g.bufferline` inside the `if` branch, that avoids copying the dict from vimscript to lua unless it's really needed.
 - On the docs examples, can you put as value `let bufferline.no_name_title = v:null`? Feel free to add a comment to explain the default behavior, and the behavior if the value is set to a string. This is for consistency with the other options which all show their default value. This allows people to just copy/paste the default config in their RC files and play with the options they want to update.

Reply:
oke wait